LittleFileSystem2.cpp

```diff
@@ -24,8 +24,9 @@ namespace mbed {
 
 extern "C" uint32_t lfs2_crc(uint32_t crc, const void *buffer, size_t size)
 {
-    uint32_t initial_xor = crc;
-    MbedCRC<POLY_32BIT_REV_ANSI, 32> ct(initial_xor, 0x0, true, false);
+    uint32_t initial_xor = lfs2_rbit(crc);
```

Comment:
@geky, @kjbracey-arm This part is like black magic for me so I trust you also take a look.

Reply:
It was taken from https://github.com/ARMmbed/mbed-os/blob/master/features/storage/filesystem/littlefs/LittleFileSystem.cpp#L27

Reply:
Which comes from this PR from @kjbracey-arm https://github.com/ARMmbed/mbed-os/pull/11957

Given that it's the same as what's on mbed-os, I think it should be fine. The CRC didn't change v1->v2.